roofit/roofitcore/src/RooAbsReal.cxx

```diff
@@ -4876,12 +4876,11 @@ RooSpan<double> RooAbsReal::evaluateBatch(std::size_t begin, std::size_t maxSize
   leafNodeServerList(&allLeafs);
 
   if (RooMsgService::instance().isActive(this, RooFit::Optimization, RooFit::INFO)) {
-    coutI(Optimization) << "The class " << IsA()->GetName() << " does not have the faster batch evaluation interface."
-          << " Consider requesting this feature on ROOT's JIRA tracker." << std::endl;
+    coutI(Optimization) << "The class " << IsA()->GetName() << " could benefit from implementing the faster batch evaluation interface."
+        << " If it is part of ROOT, consider requesting this on https://root.cern." << std::endl;
```

Comment:
```suggestion
        << " If it is part of ROOT, consider requesting this on https://sft.its.cern.ch/jira/projects/ROOT." << std::endl;
```

Reply:
I feared that the github issues migration would make this obsolete too fast. Opinions?

Reply:
imo you are right :D